ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1220,17 +1220,22 @@ private class JdbcLedgerDao(
     ~ str("observers").? map flatten)
 
   private val ContractLetParser = (ledgerString("id")
-    ~ date("effective_at").? map flatten)
+    ~ date("effective_at").?
+    ~ long("archive_offset").? map flatten)
```

Comment:
This column is not in the select clause of the query. I don't think we need to handle the archive offset outside the query.

Reply:
Ah yes we don't. Left-over from an earlier iteration.. thanks for catching!